docs/src/pages/components/Dialogs/WebDialog.mdx

```diff
@@ -0,0 +1,54 @@
+---
+title: Dialog
+platform: web
+githubPath: bpk-component-dialog
+---
+
+import Readme from 'components/Readme';
+import PresentationBlock from 'components/PresentationBlock';
+import readme from 'bpk-component-dialog/README.md';
+
+import {
+  DefaultExample,
+  WithIconExample,
+  NotDismissibleExample,
+  WithFlareExample,
+} from 'bpk-component-dialog/examples';
+
+## Table of contents
+
+## Default
+
+The default dialog gives you a blank canvas with a close icon. Tapping the scrim or close icon will dismiss it.
+
+<PresentationBlock>
+  <DefaultExample />
+</PresentationBlock>
+
+## Non dismissible
+
+In cases when you want the user to explicitly make or accept the choice, the non dissmissible configuration means the dialog has no close icon, and tapping the scrim will not dismiss it. Instead, the user must interact with the dialog&apos;s content in order to dismiss it.
```

Comment:
typo

```suggestion
In cases when you want the user to explicitly make or accept the choice, the non dismissible configuration means the dialog has no close icon, and tapping the scrim will not dismiss it. Instead, the user must interact with the dialog&apos;s content in order to dismiss it.
```

Reply:
Ooo good spot! This was wrong previously so fix at the same time as migration!